davidshare/Fast-Food-Fast
# #160634276 Setup and configure database for app

#### What does this PR do?
Setup and configure database for app

#### Description of Task to be completed?
- install node-postgres, bcrypt and dotenv
- create .env file to host environment variables
- create config directory and add config file for database configuration
- create database connection file
- create files in models directory to run database migrations
- add database migration scripts to package.json file
- refactor models index file to run database migrations

#### How should this be manually tested?
- clone the repo and navigate to the root directory using the command line
- on the command line type npm install to install the node modules
- install pg-admin 3 and create a database fastfood
- on the command line, type 'npm run db-migrate'
- open the database on pg-admin to confirm that tables have been created

#### What are the relevant pivotal tracker stories?
[#160634276](https://www.pivotaltracker.com/story/show/160634276)
